include/libpmemobj++/detail/common.hpp

```diff
@@ -101,6 +101,22 @@ type_num()
 	return typeid(T).hash_code();
 }
 
+/**
+ * Round up to the next lowest power of 2.
+ */
+inline uint64_t
```

Comment:
why uin64_t?
I'd prefer a template variant which does this based on size of the provided type.
Not sure if we could use some kind of constexpr if?